tests/sub_group_mask/sub_group_mask_reference.cpp

```diff
@@ -0,0 +1,86 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Extension Conformance Test
+//
+//  Provides tests to check sub_group_mask operator[] const and reference
+//
+*******************************************************************************/
+
+#include "sub_group_mask_common.h"
+
+#define TEST_NAME sub_group_mask_reference
+
+namespace TEST_NAMESPACE {
+
+using namespace sycl_cts;
+#ifdef SYCL_EXT_ONEAPI_SUB_GROUP_MASK
+
+struct check_result_reference {
```

Comment:
Add a high-level comment explaining what is done here in English so we can check the computation bellow.

Reply:
Done in https://github.com/KhronosGroup/SYCL-CTS/pull/201/commits/f5d957124506ba3e3f168571d8b1c08f811961ed